### PR TITLE
Add animate component

### PR DESCRIPTION
**Pull request by trenaryja**
Add Animate component to wrap content in an animating div. 

Thinly wraps the Framer-Motion library, and exposes some pre-determined animations for use within the PCLN design system.

### CODE REVIEW

**Review by craigpalermo — CHANGES_REQUESTED** [review]
The code change LGTM, but be sure to confirm that the Interactions tab is configured in the Storybook app as well.

**Review by jligi — APPROVED** [review]

**Review by ryan-hunt-priceline — APPROVED** [review]

File: packages/core/.storybook/main.js [hunk 1]
```diff
@@ -2,8 +2,11 @@ const { heftStoryGlobs, storykitAddons, features, framework, core } = require('@
 
 module.exports = {
   stories: heftStoryGlobs,
-  addons: storykitAddons,
-  features,
+  addons: [...storykitAddons, '@storybook/addon-interactions'],
+  features: {
+    ...features,
+    interactionsDebugger: true,
+  },
```
Comment (pcln-james) [inline packages/core/.storybook/main.js:9]: do we want to commit this change?

**Review by pcln-james — APPROVED** [review]
Great work! The storybook examples are great.

File: packages/core/.storybook/main.js (hunk 1 above)
Comment (trenaryja) [reply packages/core/.storybook/main.js:9]: I believe so! This configurations change allows us to perform interaction testing through storybook, which is particularly useful for animation testing!

**Review by craigpalermo — APPROVED** [review]

**Pull request merged**